PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalVaultRequest.kt

```diff
@@ -37,7 +37,7 @@ class PayPalVaultRequest
     var shouldOfferCredit: Boolean = false,
     var recurringBillingDetails: PayPalRecurringBillingDetails? = null,
     var recurringBillingPlanType: PayPalRecurringBillingPlanType? = null,
-    var enablePayPalAppSwitch: Boolean = false,
+    override var enablePayPalAppSwitch: Boolean = false,
```

Comment:
Nit - I can't highlight the line, but should we remove the `enablePayPalAppSwitch` kdoc to align with the other overridden properties?

Reply:
done